### DISCUSSION TITLE
On device compilation

### DISCUSSION THREAD

**Discussion by MaratZakirov** [termux/termux-packages#9390] [Q&A] [answered]
waiting for an hour of on device compilation and geting some weird error after makes termux completly unusable for me. Could you please give mirror with prebuild binaries?

**Comment by finagolfin** [comment]
Not sure what you're trying to do, run `pkg up` and there are already plenty of prebuilt packages.

**Reply by MaratZakirov** [reply]
Really? Prebuilt numpy scipy matplotlib?

**Reply by finagolfin** [reply]
Oh, since you didn't name what you were trying to build before, I thought you were building packages from this repo.

I think there may have been builds of those before, but maybe the guy providing them stopped doing so.

**Comment by Grimler91** [comment, marked as answer]
Matplotlib is now available from main repository. Numpy and scipy are not, and there are no repositories currently that have working scipy and/or numpy packages for termux

**Reply by MaratZakirov** [reply]
matplotlib without numpy? Is it available from your grimler termux chanel? Could you please provide working instruction for numpy + scipy + matplotlib from termux?
